### PR TITLE
Refactor to expose functions for third party use.

### PR DESCRIPTION
**Pull request by gdmen**
TODO: add a small unit test that checks for the methods & signatures that are used by cephalopod. Test fails if they are changed in a backwards incompatible way.

### CODE REVIEW

File: brkt_cli/encryptor_service.py [hunk 1]
```diff
@@ -17,6 +17,7 @@
 import logging
 import urllib2
 
+ENCRYPT_INITIALIZING = 'initial'
```
Comment (bburtin) [inline brkt_cli/encryptor_service.py]: Nit: "initial" seems inconsistent with "downloading" and "encrypting".

Comment (gdmen) [inline brkt_cli/encryptor_service.py]: That's what the encryptor instance already sends. I'm just adding client side handling for the state that we didn't have before. I'll change this on the avatar side the next time I'm making a change there. Not worth a NUC imo.

**Comment by bburtin** [comment]
Looks good.

**Comment by gdmen** [comment]
Minor fixes after testing HVM & PV encryption. Unit tests coming later today.

**Comment by gdmen** [comment]
Unit test added to check backwards compatibility with current outside usage.

File: brkt_cli/encrypt_ami.py [hunk 2]
```diff
@@ -713,9 +717,229 @@ def log_exception_console(aws_svc, e, id):
         )
 
 
+def get_encrypted_ami_kwargs(aws_svc, enc_svc_cls, encryptor_instance,
```
Comment (bburtin) [inline brkt_cli/encrypt_ami.py]: I'm not sure about the kwargs thing.  kwargs make it hard to know what is being returned by the first function, and what's being passed into the second.  Can this function return a tuple instead?

**Comment by bburtin** [comment]
Looks good.  Ship it!

**Pull request merged**